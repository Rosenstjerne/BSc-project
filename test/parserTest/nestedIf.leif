var int a,b;
a = 1;
b = 2;

if (a == 1)
    if (b == 2) 
        print(a+b);

if (2 == b) {
    if (1 == a) {
        print(b-a);
    }
}
