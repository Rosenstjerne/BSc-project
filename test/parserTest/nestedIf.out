function null main() {
	var int a, b;
	
	a = 1;
	b = 2;
	if((a == 1))
	#begin if
		if((b == 2))
		#begin if
			print((a + b));
		#end if
	#end if
	if((2 == b))
	#begin if
		if((1 == a))
		#begin if
			print((b - a));
			
		#end if
		
	#end if
	
}
